### DISCUSSION TITLE
Upgrade api version of Kustomization and Helmrelease to v1

### DISCUSSION THREAD

**Discussion by torbjornvatn** [fluxcd/flux2#4847] [Q&A] [answered]
We have just done an upgrade from a really ancient version of flux and the old kind of terraform-provider to flux v2.3.0 and v1.3 of the provider.

Everything seems to work as it should, but we are not able to upgrade the api versions of our Kustomizations and Helmreleases.
We have updated our manifests to have the new `/v1` versions, but it's not reflected in the resources found in the cluster, they're still on the `/v2beta1` versions.
And I can't find any way to make them reconcile in a way that updates the api versions.

Does anyone have any tips on how to fix this?

**Comment by stefanprodan** [comment, marked as answer]
> We have updated our manifests to have the new /v1 versions, but it's not reflected in the resources found in the cluster, they're still on the /v2beta1 versions.

Try deleting you local kubectl cache, if all resources in Git are on the latest version and they were reconciled, then this is just a local cache issue.
